app/main/db/queries.py

```diff
@@ -19,43 +19,44 @@ def fuzzy_search(query: str, page: int, per_page: int):
 def browse_data(
     page,
     per_page,
+    browse_type="browse",
+    filters=None,
+    sorting_orders=None,
     transferring_body_id=None,
     series_id=None,
     consignment_id=None,
-    date_range=None,
-    date_filter_field=None,
 ):
-    if transferring_body_id:
+    if browse_type == "transferring_body":
```

Comment:
Nice change.

Just rehighlighting the point made in slack yesterday about the urls. I think in the next sprint we can start to split out the browse views into their own distinct routes,e.g.

```
@route("/browse")
def browse():
```


```
@route("/browse/consignment/<uuid>:consignment_id")
def browse_consignment(consignment_id):
```

```
@route("/browse/series/<uuid>:series_id")
def browse_consignment(series_id):
```

```
@route("/browse/transferring_body/<uuid>:transferring_body_id")
def browse_transferring_body(transferring_body_id):
```

so we have 
```
/browse/
/browse/transferring_body/id
/browse/consignment/id
/browse/series/id
```


or even 

```
/browse/
/transferring_body/id/browse
/consignment/id/browse
/series/id/browse
```

could even be better 

but yeah, should be easy to break this function out into each of those when we create those tickets.